File: objectorientedprogramming/oops/stud.py
class student:
    def set_student(self, rol, name, course):
        self.rol = rol
        self.name = name
        self.course = course

    def get_student(self):
        print(self.rol, ",", self.name, ",", self.course)


obj = student()
obj.set_student(10, "engineering", 'jyothi')
obj.get_student()
# print(obj.course)
# print(obj.rol)

File: javascript/objectoriented/stud.js
class Student{
    setStudent(rolno,name,course){
        this.rolno=rolno;
        this.name=name;
        this.course=course;
    }
    printStudent(){
        console.log(this.rolno+","+this.name+","+this.course);
    }
}
//object
//referencename=className()
var obj=new Student();
obj.setStudent(100,"ajay","mca")
obj.printStudent()
